rero_ils/modules/acq_order_lines/dumpers.py

```diff
@@ -66,3 +67,29 @@ def dump(self, record, data):
         }
         data['document'] = {k: v for k, v in data['document'].items() if v}
         return data
+
+
+class AcqOrderLineNotificationDumper(InvenioRecordsDumper):
+    """Order line dumper class for acquisition order."""
+
+    def dump(self, record, data):
+        """Dump an AcqOrderLine instance for acquisition order notification.
+
+        :param record: The record to dump.
+        :param data: The initial dump data passed in by ``record.dumps()``.
+        """
```

Comment:
If the data is the `record.dumps()`, why we need the data we can do it here:
`data = records.dumps()` ?